php/pantheon/utils.php

```diff
@@ -62,6 +62,26 @@ public static function sanitize_data($data, $sanitizer_function = 'htmlspecialch
 			return is_object( $data ) ? (object)$sanitized_data : $sanitized_data;
 		} elseif ( is_integer($data) ) {
 			return (string)$data;
+		} elseif ( is_string( $data ) ) {
+			if ( ! empty( $data ) ) {
+				$sanitized_data = $sanitizer_function($data);
+				$dom = new \DOMDocument;
+				$dom->loadHTML( $sanitized_data );
+				$anchors = $dom->getElementsByTagName('a');
+
+				// Bail if our string does not only contain an anchor tag.
+				if ( 0 === $anchors->length ) {;
```

Comment:
```suggestion
				if ( 0 === $anchors->length ) {
```
is this a typo? Valid just unconventional?